packages/bpk-component-image/readme.md

```diff
@@ -13,7 +13,7 @@ npm install bpk-component-image --save-dev
 ```js
 import React from 'react';
 import BpkImage from 'bpk-component-image';
-import * as BREAKPOINTS from 'bpk-tokens/tokens/breakpoints.es6';
+import { breakpointDesktop, breakpointTablet } from 'bpk-tokens/tokens/base.es6';
```

Comment:
The reason we added `breakpoints.es6` was for consumers who didn't yet have tree-shaking but wanted to use the `BpkBreakpoint` component in conjunction with the breakpoint tokens. This is less of an issue now as the majority of consumers are now on a version of `backpack-react-scripts` that supports tree-shaking.